client/components/AddCaptionForm.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+import PostImage from './PostImage'
+
+export default React.createClass({
+  render () {
+    return (
+      <div className="container">
+        <div className="row">
+          <PostImage />
+        </div>
+        <div className="row">
+          <div className="form-group">
+            <label htmlFor="exampleTextarea">Drop a line down below</label>
+            <textarea className="form-control" id="exampleTextarea" rows="3"></textarea>
+            <a href="#">
```

Comment:
Do we need to wrap this button with an anchor tag? Once we add props, we can put the event handler on the button.